src/EventListener/SerializeListener.php

```diff
@@ -61,15 +61,8 @@ public function onKernelView(GetResponseForControllerResultEvent $event): void
 
         $context = $this->serializerContextBuilder->createFromRequest($request, true, $attributes);
 
-        if (
-            (isset($context['output']) && \array_key_exists('class', $context['output']) && null === $context['output']['class'])
-            ||
-            (
-                null === $controllerResult && isset($context['input']) && \array_key_exists('class', $context['input']) &&
-                null === $context['input']['class']
-            )
-        ) {
-            $event->setControllerResult('');
+        if (isset($context['output']) && \array_key_exists('class', $context['output']) && null === $context['output']['class']) {
```

Comment:
maybe 

```suggestion
        if (\array_key_exists('class', $context['output'] ?? []) && null === $context['output']['class']) {
```

even if it is not directly linked to your change

Reply:
I'd be to keep isset, it's readable and doesn't harm no?